components/GlobalNewsAndUpdates.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import NewsAndUpdatesModal from './NewsAndUpdatesModal';
+import { withNewsAndUpdates } from './NewsAndUpdatesProvider';
+
+const GlobalNewsAndUpdates = props => {
+  const { showNewsAndUpdates, setShowNewsAndUpdates } = props;
```

Comment:
Ah thanks. The way I thought was since we have to have, `withNewsAndUpdates` due to being used in the class component `TopBarProfileMenu` adding a hook to do the same thing might be redundant. 😄